composer.json

```diff
@@ -118,7 +118,7 @@
         "predis/predis": "~1.1",
         "psr/http-client": "^1.0",
         "psr/simple-cache": "^1.0",
-        "egulias/email-validator": "~1.2,>=1.2.8|~2.0",
+        "egulias/email-validator": "^1.2.8|^2|^3",
```

Comment:
Shall I change bump to `^2.1.10|^3` here to be consistent with the constraint in the components? I don't think that we're testing with v1 anywhere at the moment.

Reply:
Let's drop 1.2, no need to support 3 major versions anyway.

Reply:
Do we really want to drop support for it in a patch release?

Reply:
We should drop it in 5.x only.